.github/workflows/firefox-ios-ui-tests.yml

```diff
@@ -68,6 +68,23 @@ jobs:
               run: |
                 gem install xcpretty -v 0.3.0
                 pip install blockkit==1.9.1
+            - name: Setup Xcode
```

Comment:
Is not this included in this PR https://github.com/mozilla-mobile/firefox-ios/pull/20990/files?

let's try to consolidate changes if possible

Reply:
Yes, let me close this PR because it is not needed anymore.